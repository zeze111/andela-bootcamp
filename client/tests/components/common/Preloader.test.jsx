import React from 'react';

import Preloader from '../../../components/common/Preloader.jsx';

describe('Test for Preloader', () => {
  it('should render component', () => {
    const wrapper = shallow(<Preloader /> );
    expect(wrapper).toMatchSnapshot()
  });
});
