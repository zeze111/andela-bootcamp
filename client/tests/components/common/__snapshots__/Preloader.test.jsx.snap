// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for Preloader should render component 1`] = `
<div
  className="preloader-wrapper small active"
>
  <div
    className="spinner-layer spinner-blue"
  >
    <div
      className="circle-clipper left"
    >
      <div
        className="circle"
      />
    </div>
    <div
      className="gap-patch"
    >
      <div
        className="circle"
      />
    </div>
    <div
      className="circle-clipper right"
    >
      <div
        className="circle"
      />
    </div>
  </div>
  <div
    className="spinner-layer spinner-red"
  >
    <div
      className="circle-clipper left"
    >
      <div
        className="circle"
      />
    </div>
    <div
      className="gap-patch"
    >
      <div
        className="circle"
      />
    </div>
    <div
      className="circle-clipper right"
    >
      <div
        className="circle"
      />
    </div>
  </div>
  <div
    className="spinner-layer spinner-yellow"
  >
    <div
      className="circle-clipper left"
    >
      <div
        className="circle"
      />
    </div>
    <div
      className="gap-patch"
    >
      <div
        className="circle"
      />
    </div>
    <div
      className="circle-clipper right"
    >
      <div
        className="circle"
      />
    </div>
  </div>
  <div
    className="spinner-layer spinner-green"
  >
    <div
      className="circle-clipper left"
    >
      <div
        className="circle"
      />
    </div>
    <div
      className="gap-patch"
    >
      <div
        className="circle"
      />
    </div>
    <div
      className="circle-clipper right"
    >
      <div
        className="circle"
      />
    </div>
  </div>
</div>
`;
